spec/config.json

```diff
@@ -196,6 +196,19 @@
         "test"
       ],
       "cron": "* * * * *"
+    },
+    "hooked": {
+      "command": "echo FAIL && exit 1",
+      "subscribers": [
+        "test"
+      ],
+      "hooks": {
+          "warning": {
+              "command": "echo HOOKED",
+              "timeout": 5
+          }
+      },
```

Comment:
Note that the commit message is describing a different config format.

Also, a default timeout should probably be enforced or the attribute made mandatory.
If "Spawn.process" does already enforce something then it should probably be mentioned in the comments/docs

Reply:
We can introduce a default timeout for hooks, as it will not break backwards compatibility 👍

Reply:
@davideauee update the PR to set/use a default hook timeout of 60 seconds.